Form1.cs

```diff
@@ -103,14 +103,14 @@ public partial class Form1 : Form {
 
 			File.WriteAllText(p, acf);
 
-			MessageBox.Show("Patch applied. If everything worked correctly Steam should most likely not ask you to update Beat Saber any more. If it still does its probably related to DLC(Assmung you have any) and should not update the game, just to be sure I advise you to backup your game version", "Success");
+			MessageBox.Show("Patch applied. If everything worked out correctly, Steam should most likely not ask you to update Beat Saber anymore. If it still does, this may be related to DLC (assuming you have any) and should not update the game. Be sure to backup your game version.", "Success");
 		}
 
 		private void aboutButton_Click(object sender, EventArgs e) {
 			MessageBox.Show(
-				"This tool modifies a config file of Steam to make it think that you already have the last version (Indicated by the correct Manifest ID) eventho you are not.\n" +
+				"This tool modifies a configuration file of Steam to spoof the version Saber Version to be the latest without actually updating your local version.\n" +
```

Comment:
I worded it simply on purpose. Not everyone might have an idea what "to spoof" means